code/modules/item_worth/worths_list.dm

```diff
@@ -710,5 +710,7 @@ var/list/worths = list(
 					/obj/machinery/libraryscanner = -1000,
 					/obj/machinery/bookbinder = -1200,
 					/obj/machinery = -1000,
-					) //Must be in descending order. Child before parents, otherwise it doesn't work.,
-
+// BOOKS
+					/obj/item/book/skill = 60 // individual skill books should PROBABLY be priced according to skill level... but im lazy
```

Comment:
Comments within lists sometimes will show up as false positives on whitespace tests. I believe terminating the comment with `,` can make it pass, or possibly some combination of newline and tab.

Reply:
Oh I see. Thank you! I didn't see this before a87935f but if it doesn't work I'll try that.